cloud/migrate-to-cloud/hypershift.md

```diff
@@ -258,19 +258,26 @@ include_schemas:
 - '"FooBar"'
```

Comment:
Can we be more specific here? What are people expected to have here? Perhaps `'"<SCHEMA_NAME>"'` ?